gstd/src/msg/async.rs

```diff
@@ -61,6 +61,15 @@ impl<D: Decode> FusedFuture for CodecMessageFuture<D> {
     }
 }
 
+/// To interrupt a program execution waiting for a reply on a previous message,
+/// one need to call an `.await` expression.
+/// The initial message that requires a reply is sent instantly.
+/// Methods `send_and_wait_for_reply` return `MessageFuture` where `.await` can
```

Comment:
@breathx please check if description is ok

Reply:
I've already commented on this description. I still think that formulation like "return `MessageFuture` which implements `Future` trait" is more correct neither than given "where `.await` can be called", because Rust's async is common and understandable thing

Probably it will be more clearly if we describe with words return types instead of directly point at `Result` variants, but I doubt is it better for documentation.

I also suppose that "if reply was received" is a bit complete explanation than "if reply exists"

Reply:
updated